task/backend/scheduler.go

```diff
@@ -41,6 +41,9 @@ type RunCreation struct {
 	// Unix timestamp for when the next run is due.
 	NextDue int64
 
+	// time.Time for when the next run is due (includes offset)
+	RunAt time.Time
+
```

Comment:
This is not necessary, as you'll be removing the `RunCreation` type with your cleanup PR